ads/vendors/digiteka.md

```diff
@@ -0,0 +1,40 @@
+<!---
+Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Digiteka
+
+## Example
+
+### Basic
+
+```html
+<amp-ad
+  width="300"
+  height="1"
+  type="digiteka"
+  layout="responsive"
+  data-mdtk="01212121"
+>
+</amp-ad>
+```
+
+## Configuration
+
+For configuration semantics, please [contact Digiteka](http://digiteka.com).
+
+Supported parameters:
+
+-   `src`
```

Comment:
I don't see src in your example? Also could you add `data-zone` & `data-mdtk`? Thanks

Reply:
Modification has been made